modules/platforms/cpp/client/include/ignite/ignite_client_configuration.h

```diff
@@ -85,10 +80,7 @@ class IgniteClientConfiguration
      *
      * @return Current logger.
      */
-    [[nodiscard]]
-    const std::shared_ptr<IgniteLogger>& getLogger() const {
-        return m_logger;
-    }
+    [[nodiscard]] const std::shared_ptr<IgniteLogger> &getLogger() const { return m_logger; }
```

Comment:
I'm not sure about putting function attributes and body on the same line as other declaration by default - it feels to me as there is too much on a single line and it's harder to read such a code. Also, this leaves even less space for arguments.

Reply:
Unfortunately clang-format doesn't have an option for this. Or at least I was not able to find it. We either use it as is or give up and do everything manually. I hope they will add such an option in future.